asyncapi/asgardeo/data_types.bal

```diff
@@ -52,6 +52,27 @@ public type GenericUserData record {
     map<string> claims?;
 };
 
+# Update User Credentials Data
+#
+# + organizationId - Organization Id  
+# + ref - Reference
+# + organizationName - Organization Name  
+# + userStoreName - Name of the User Store
+# + userName - User Name
+# + userId - User Id
+# + scenario - Credential Updated scenario
+# + claims - Claims
+public type UpdateUserCredentialsData record {
+    int organizationId?;
+    string ref?;
+    string organizationName?;
+    string userStoreName?;
+    string userName?;
+    string userId?;
+    string scenario?;
+    map<string> claims?;
```

Comment:
Marking a field with `?` makes that field optional. Can you share an expected sample payload of this type ?